### PR TITLE
Change datatable field names

### PR DESCRIPTION
**Pull request by hutnikau**
Please merge https://github.com/oat-sa/extension-tao-proctoring/pull/277 before

### CODE REVIEW

File: controller/Monitor.php [hunk 1]
```diff
@@ -79,11 +79,12 @@ public function index()
      */
     public function deliveryExecutions()
     {
-        $requestOptions = $this->getRequestOptions(['sortby' => 'date', 'sortorder' => 'desc']);
+        $requestOptions = $this->getRequestOptions(['sortBy' => 'date', 'sortOrder' => 'DESC']);
```
Comment (antoinerobin) [inline controller/Monitor.php]: case didn't change in SimplePageModule so keep ```sortby```and ```sortorder```

File: controller/Monitor.php [hunk 2]
```diff
@@ -79,11 +79,12 @@ public function index()
...
+        $options['order'] = $requestOptions['sortBy'] . ' ' . mb_strtoupper($requestOptions['sortOrder']);
         $service = $this->getServiceManager()->get(ProctorService::SERVICE_ID);
         $proctor = \common_session_SessionManager::getSession()->getUser();
         $delivery = $this->getCurrentDelivery();
-        $executions = $service->getProctorableDeliveryExecutions($proctor, $delivery);
+        $executions = $service->getProctorableDeliveryExecutions($proctor, $delivery, null, $options);
```
Comment (antoinerobin) [inline controller/Monitor.php]: getProctorableDeliveryExecutions allows only 3 paraleters ... maybe you forget to commit/push a modification to this method

**Comment by antoinerobin** [comment]
This works for all "primary" columns such as date but not for kv as last or first name

**Comment by hutnikau** [comment]
@antoinerobin I have made changes according to your comment and resolved the conflicts. Please review it again.

**Pull request merged**